src/livestreamer/plugins/nrk.py

```diff
@@ -10,26 +11,31 @@
     "preferred-player-live=hlslink"
 )
 
+_id_re = re.compile("/(?:program|direkte)/([^/]+)")
```

Comment:
```_id_re``` doesn't match TV series URLs, e.g.:
https://tv.nrk.no/serie/side-om-side/MUHH48000115/sesong-3/episode-1
Changing the regex string to ```"/(?:program|direkte|serie/(?:[^/]+))/([^/]+)"``` works.